src/algebra/ring/basic.lean

```diff
@@ -855,6 +860,11 @@ protected def function.surjective.non_assoc_ring
 { .. hf.add_comm_group f zero add neg sub nsmul gsmul, .. hf.mul_zero_class f zero mul,
   .. hf.distrib f add mul, .. hf.mul_one_class f one mul }
 
+lemma sub_one_mul (a b : α) : (a - 1) * b = a * b - b :=
```

Comment:
Probably worth having `one_sub_mul` too

Reply:
should I go full ham and go for `one_add_mul` as well? or is that too much